src/ebay-dialog-base/components/dialogBase.tsx

```diff
@@ -58,7 +58,7 @@ export const DialogBase: FC<DialogBaseProps<HTMLElement>> = ({
     buttonPosition = 'left',
     children,
     ariaLabelledby,
-    a11yCloseText,
```

Comment:
There was a discussion years ago why the eBay UI Core Marko try to avoid having default accessibility text. I could not find the link anymore but the problem was:

- Having the default text in English might be problematic for non-English page. For example, on German pages (`<html lang="de">`), the screenreader will read all text in German, so the English text might be read as if they're German words, which might confuse users. Not sure if this is less accessible compare to not having accessible text at all.
- To make it worse, these default english accessible text is hard to detect with accessibility checker (e.g Google Lighthouse). If the label is missing, the tool can easily point out the problems.

IMO, it's the app/consumer responsible to provide localized accessible label/text. So we should not add default text for that.

There was a discussion to make a11yText in eBay UI Core Marko mandatory, which will failed Marko template compilation, but not sure if it's implemented or not.

Reply:
Maybe we can make it non-optional for a11yCloseText. but this is only useful if the FE app uses TypeScript. But this is a good start imo.

Reply:
got it, thanks Abi!

Reply:
fixed